amplify/backend/function/getUserReputation/src/index.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
I think we to think about putting a eslint config file at the `/amplify/backend/function/` level so that it has rules just for the contained files.

Otherwise we'll just litter the lambda function code full of `//eslint-disable`'s